src/Deferred.php

```diff
@@ -14,15 +14,10 @@
 final class Deferred implements Promise
 {
     private $value;
-
```

Comment:
imho it would make sense to have phpdoc for the types of all fields here.

Reply:
Thank you